tools/pkg/clispec/clispec.go

```diff
@@ -186,6 +186,7 @@ func (e *Entity) scaffoldingKindName() string {
 	nonPluralSingletons := map[string]bool{
 		"OtelMetricsIngestion": true,
 		"LogIngestConfig":      true,
+		"LogControlConfig":     true,
```

Comment:
Oh, I think we can just get rid of this whole thing. No singleton is plural -- I'm not sure why this allowlist exists.

Reply:
how is it used? is it?